app/presenters/work_presenter.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# Presents a work
+class WorkPresenter < ApplicationPresenter
+  def initialize(work:, work_form:)
+    @work = work
+    @work_form = work_form
+    super()
+  end
+
+  attr_reader :work, :work_form
+
+  delegate :status_message, :editable?, to: :status_presenter
+  delegate :collection, :created_at, :druid, :user, to: :work
+  delegate :abstract, :contact_emails_attributes, :keywords_attributes, :license, :other_work_subtype,
+           :related_links_attributes, :related_works_attributes, :title, :version, :work_subtypes, :work_type,
+           to: :work_form
+
+  def purl_link
+    # No druid yet, so there's no PURL link yet either. Work is likely still depositing.
+    return if druid.blank?
+
+    link_to(nil, Sdr::Purl.from_druid(druid:))
+  end
+
+  def contact_emails
+    contact_emails_attributes.map(&:email).join(', ')
+  end
+
+  def collection_link
+    link_to(collection.title, Rails.application.routes.url_helpers.collection_path(collection))
```

Comment:
Do you need "Rails.application.routes.url_helpers" since it is included in ApplicationPresenter?

Reply:
I was surprised to find that, yes, I did.